automation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:15.04
+
+# Update index
+RUN apt-get update && apt-get -y upgrade
```

Comment:
It would probably be better to remove this and instead use `docker build --pull ...` to build the image, ie make sure we're using the latest version of the `ubuntu:15.04` image